autopush/scripts/drop_user.py

```diff
@@ -0,0 +1,39 @@
+import time
+
+import click
+
+from autopush.db import (
+    get_router_table,
+    Router,
+)
+from autopush.metrics import SinkMetrics
+
+
+@click.command()
+@click.option('--router_table_name', help="Name of the router table.")
+@click.option('--months-ago', default=2, help="Months ago to remove.")
+@click.option('--batch_size', default=25,
+              help="Deletes to run before pausing.")
+@click.option('--pause_time', default=1,
+              help="Seconds to pause between batches.")
+def drop_users(router_table_name, months_ago, batch_size, pause_time):
+    router_table = get_router_table(router_table_name)
+    router = Router(router_table, SinkMetrics())
+
+    click.echo("Deleting users with a last_connect %s months ago."
+               % months_ago)
+
+    count = 0
+    for deletes in router.drop_old_users(months_ago):
+        click.echo("")
+        count += deletes
+        if count > batch_size:
```

Comment:
Guess I'm confused by this function then. 

If there's 100 records to be deleted, we fire off two sets of 25, sleep a second, then fire of the second 2 sets of 25, correct?

We only reset count to zero if we've exceeded 25 records, and we always increment record counts by a max of 25, or am I missing something key?